src/controller/src/clusters.rs

```diff
@@ -625,12 +628,39 @@ where
         let aws_connection_role_arn = self.connection_context().aws_connection_role_arn.clone();
         let persist_pubsub_url = self.persist_pubsub_url.clone();
         let secrets_args = self.secrets_args.to_flags();
+
+        // TODO(teskje): use the same values as for compute?
+        let storage_proto_timely_config = TimelyConfig {
+            // This value is not currently used by storage, so we just choose
+            // some identifiable value.
+            arrangement_exert_proportionality: 1337,
+            ..Default::default()
+        };
```

Comment:
It would be really easy to use the same config as compute here. Still deferring to a follow-up since it's a behavior change.